---
title: "A reference map of the human binary protein interactome"
category: articles
permalink: /articles/2020-04-08-huri-nature/ 
excerpt: "Global insights into cellular organization and genome function require comprehensive understanding of the interactome networks that mediate genotype–phenotype relationships1,2. Here we present a human ‘all-by-all’ reference interactome map of human binary protein interactions, or ‘HuRI’. With approximately 53,000 protein–protein interactions, HuRI has approximately four times as many such interactions as there are high-quality curated interactions from small-scale studies. The integration of HuRI with genome3, transcriptome4 and proteome5 data enables cellular function to be studied within most physiological or pathological cellular contexts. We demonstrate the utility of HuRI in identifying the specific subcellular roles of protein–protein interactions. Inferred tissue-specific networks reveal general principles for the formation of cellular context-specific functions and elucidate potential molecular mechanisms that might underlie tissue-specific phenotypes of Mendelian diseases. HuRI is a systematic proteome-wide reference that links genomic variation to phenotypic outcomes."
date: 2020-04-08
venue: 'Nature
---

<a href='https://doi.org/10.1038/s41586-020-2188-x'>Link to the open access article here</a>. This paper is part of the ECSCW 2018 conference and is published in the Journal of Computer-Supported Cooperative Work. The paper won an honourable mention for best paper at ECSCW 2018, and the presentation of the paper won best presentation at the conferece.

Abstract: Global insights into cellular organization and genome function require comprehensive understanding of the interactome networks that mediate genotype–phenotype relationships1,2. Here we present a human ‘all-by-all’ reference interactome map of human binary protein interactions, or ‘HuRI’. With approximately 53,000 protein–protein interactions, HuRI has approximately four times as many such interactions as there are high-quality curated interactions from small-scale studies. The integration of HuRI with genome3, transcriptome4 and proteome5 data enables cellular function to be studied within most physiological or pathological cellular contexts. We demonstrate the utility of HuRI in identifying the specific subcellular roles of protein–protein interactions. Inferred tissue-specific networks reveal general principles for the formation of cellular context-specific functions and elucidate potential molecular mechanisms that might underlie tissue-specific phenotypes of Mendelian diseases. HuRI is a systematic proteome-wide reference that links genomic variation to phenotypic outcomes.



Recommended citation: Luck, K., Kim, DK., Lambourne, L. et al. A reference map of the human binary protein interactome. Nature 580, 402–408 (2020). https://doi.org/10.1038/s41586-020-2188-x


